keys.go

```diff
@@ -38,6 +38,17 @@ var (
 // PreferReturn designates the value for the "Prefer" header.
 type PreferReturn int
 
+// EncryptionAlgorithm represents the encryption algorithm used for key creation
+type EncryptionAlgorithm string
```

Comment:
Lets not do a superfluous type alias, and just use string constants... it saves a confusing cast later as well.

Reply:
Done.